templates/base.html.twig

```diff
@@ -8,9 +8,47 @@
         {% endblock %}
     </head>
     <body>
-        {% include "include/header.html.twig" %}
+        <header class="header">
+            <nav class="navbar navbar-expand-lg navbar-dark justify-content-between">
+                <a class="navbar-brand" href="/">
+                    <img src="{{ asset('images/mvp-logo-light.png' ) }}"
+                            width="50"
+                            height="50"
+                            alt="MVP Office Supplies Logo"
+                    >
+
+                    <span class="p-3">MVP Office Supplies</span>
+                </a>
+
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        {% if is_granted('IS_AUTHENTICATED_REMEMBERED') %}
+                            <a class="nav-link" href="{{ path('app_logout') }}">
+                                Log out
+                            </a>
+                        {% else %}
+                            <a class="nav-link" href="{{ path('app_login') }}">
+                                Log in
```

Comment:
I'm concerned here about adding functionality that we're not going to need nor talk about during the entirety of the tutorial. And also, don't you think templates look cleaner and better if you extract header/footer functionality into its own files?

Reply:
> I'm concerned here about adding functionality that we're not going to need nor talk about during the entirety of the tutorial.

That's fair. We'll probably need these eventually (as a way to log in, but maybe not in this tutorial) and they *do* show that this is a true, multi-page app.

> And also, don't you think templates look cleaner and better if you extract header/footer functionality into its own files?

Nope :). But that is *totally* 100% subject, I realize :). I've not seen anyone do this before, unless those template partials were being reused.